xbmc/guilib/guiinfo/GUIInfoLabels.h

```diff
@@ -593,6 +593,7 @@
 #define PVR_TIMESHIFT_PROGRESS_START_TIME (PVR_STRINGS_START + 70)
 #define PVR_TIMESHIFT_PROGRESS_END_TIME   (PVR_STRINGS_START + 71)
 #define PVR_STRINGS_END                   PVR_TIMESHIFT_PROGRESS_END_TIME
+#define PVR_EPG_EVENT_ICON                (PVR_STRINGS_START + 72)
```

Comment:
Move this line above the line containing PVR_STRINGS_END, adapt that line to: `#define PVR_STRINGS_END PVR_EPG_EVENT_ICON `. You get the gist of this?